hermes-management/src/main/java/pl/allegro/tech/hermes/management/config/ManagementConfiguration.java

```diff
@@ -42,7 +43,7 @@ public ObjectMapper objectMapper() {
         mapper.setSerializationInclusion(JsonInclude.Include.NON_NULL);
         mapper.disable(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES);
         mapper.disable(SerializationFeature.WRITE_NULL_MAP_VALUES);
-        mapper.registerModule(new JavaTimeModule());
+        mapper.registerModules(new JavaTimeModule(), new Jdk8Module());
```

Comment:
can you add a short  code comment why do we need to add this `Jdk8Module()` ?

Reply:
Sure on it